### DISCUSSION TITLE
Client bundle size

### DISCUSSION THREAD

**Discussion by thomasverleye** [sergiodxa/remix-i18next#122] [Q&A]
I've noticed in pagespeed that the bundle size of my `entry.client-XXXX.js` was 300KB.
When using [bundle-wizard](https://www.npmjs.com/package/bundle-wizard) for analyzation with sourcemaps enabled, I've noticed something odd.

The script does contain i18next, but it seems to be loading `cjs` AND `esm`, which both has a size of ~56KB.
<img width="1552" alt="Screenshot 2023-03-04 at 15 55 37" src="https://user-images.githubusercontent.com/2844485/222912908-82041340-3fa1-429e-a23f-708d7e424e83.png">

Now I do know this seems like this is something for the i18next lib but I just wanted to check if anyone else here had any issues with this as well.

Here are all modules I'm using on my current setup:
```
    "i18next": "^22.4.9",
    "i18next-fs-backend": "^2.1.1",
    "i18next-http-backend": "^2.1.1",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-i18next": "^12.1.5",
    "remix-i18next": "^5.0.0",
```

**Comment by sergiodxa** [comment]
Most likely you’re seeing a mixed bundle with client and server code, I also see remix-i18next server.js file.

Ensure you’re using the source maps of the client build and not the server build, if you still get it try to raise an issue with i18next itself.
